src/compass/utils/lut.py

```diff
@@ -0,0 +1,22 @@
+def compute_lut(burst, xstep=500, ystep=50):
+    '''
+    Compute slant range and azimuth LUTs corrections
+    to be applied during burst geocoding
+
+    Parameters
+    ----------
+    burst: Sentinel1BurstSlc
+        S1-A/B burst object
+    xstep: int
+        LUT spacing (in pixels) along x/slant range
+    ystep: int
+        LUT spacing (in pixels) along x/azimuth
```

Comment:
```suggestion
        LUT spacing (in pixels) along x/slant range in meters
    ystep: int
        LUT spacing (in pixels) along x/azimuth in seconds
```
Added units to be consistent with implementation in `isce3::geocode::geocodeSlc`.